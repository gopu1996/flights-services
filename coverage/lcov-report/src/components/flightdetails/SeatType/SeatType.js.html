<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/flightdetails/SeatType/SeatType.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="..\..\..\..\prettify.css" />
    <link rel="stylesheet" href="..\..\..\..\base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(..\..\..\..\sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="..\..\..\..\index.html">All files</a> / <a href="index.html">src/components/flightdetails/SeatType</a> SeatType.js
    </h1>
    <div class='clearfix'>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from "react";
&nbsp;
const SeatType = (props) =&gt; {
  const checkinPassenger = props.checkedInPassengerDetails;
  return (
    &lt;div
      className="row"
      style={{ width: "310px", position: "fixed", marginTop: "3px" }}
    &gt;
      &lt;div
        className="col"
        style={checkinPassenger ? <span class="branch-0 cbranch-no" title="branch not covered" >{ display: "" } </span>: { display: "none" }}
      &gt;
        &lt;button className="btn btn-link" disabled={true}&gt;
          &lt;i
            className="fas fa-couch"
            style={{ color: "#B3B6B7", fontSize: "20px", marginLeft: "-10px" }}
          &gt;&lt;/i&gt;
        &lt;/button&gt;
        &lt;label&gt;Available&lt;/label&gt;
      &lt;/div&gt;
      &lt;div
        className="col"
        style={checkinPassenger ? <span class="branch-0 cbranch-no" title="branch not covered" >{ display: "" } </span>: { display: "none" }}
      &gt;
        &lt;button className="btn btn-link" disabled={true}&gt;
          &lt;i
            className="fas fa-couch"
            style={{ color: "red", fontSize: "20px" }}
          &gt;&lt;/i&gt;
        &lt;/button&gt;
        &lt;label&gt;Booked&lt;/label&gt;
      &lt;/div&gt;
      &lt;div className="w-100"&gt;&lt;/div&gt;
      &lt;div
        className="col"
        style={checkinPassenger ? <span class="branch-0 cbranch-no" title="branch not covered" >{ display: "" } </span>: { display: "none" }}
      &gt;
        &lt;button className="btn btn-link" disabled={true}&gt;
          &lt;i
            className="fas fa-couch"
            style={{ color: "blue", fontSize: "20px", marginLeft: "-10px" }}
          &gt;&lt;/i&gt;
        &lt;/button&gt;
        &lt;label&gt;WheelChair&lt;/label&gt;
      &lt;/div&gt;
      &lt;div
        className="col"
        style={checkinPassenger ? <span class="branch-0 cbranch-no" title="branch not covered" >{ display: "" } </span>: { display: "none" }}
      &gt;
        &lt;button className="btn btn-link" disabled={true}&gt;
          &lt;i
            className="fas fa-couch"
            style={{ color: "orange", fontSize: "20px" }}
          &gt;&lt;/i&gt;
        &lt;/button&gt;
        &lt;label&gt;Infant&lt;/label&gt;
      &lt;/div&gt;
      &lt;div
        className="col"
        style={!checkinPassenger ? { display: "" } : <span class="branch-1 cbranch-no" title="branch not covered" >{ display: "none" }}</span>
      &gt;
        &lt;button className="btn btn-link" disabled={true}&gt;
          &lt;i
            className="fas fa-couch"
            style={{ color: "brown", fontSize: "20px" }}
          &gt;&lt;/i&gt;
        &lt;/button&gt;
        &lt;label&gt;Meals&lt;/label&gt;
      &lt;/div&gt;
    &lt;/div&gt;
  );
};
export default SeatType;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Aug 28 2020 15:53:54 GMT+0530 (India Standard Time)
</div>
</div>
<script src="..\..\..\..\prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="..\..\..\..\sorter.js"></script>
</body>
</html>
