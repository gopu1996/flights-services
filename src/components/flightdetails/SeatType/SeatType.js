import React from "react";

const SeatType = (props) => {
  const checkinPassenger = props.checkedInPassengerDetails;
  return (
    <div
      className="row"
      style={{ width: "310px", position: "fixed", marginTop: "3px" }}
    >
      <div
        className="col"
        style={checkinPassenger ? { display: "" } : { display: "none" }}
      >
        <button className="btn btn-link" disabled={true}>
          <i
            className="fas fa-couch"
            style={{ color: "#B3B6B7", fontSize: "20px", marginLeft: "-10px" }}
          ></i>
        </button>
        <label>Available</label>
      </div>
      <div
        className="col"
        style={checkinPassenger ? { display: "" } : { display: "none" }}
      >
        <button className="btn btn-link" disabled={true}>
          <i
            className="fas fa-couch"
            style={{ color: "red", fontSize: "20px" }}
          ></i>
        </button>
        <label>Booked</label>
      </div>
      <div className="w-100"></div>
      <div
        className="col"
        style={checkinPassenger ? { display: "" } : { display: "none" }}
      >
        <button className="btn btn-link" disabled={true}>
          <i
            className="fas fa-couch"
            style={{ color: "blue", fontSize: "20px", marginLeft: "-10px" }}
          ></i>
        </button>
        <label>WheelChair</label>
      </div>
      <div
        className="col"
        style={checkinPassenger ? { display: "" } : { display: "none" }}
      >
        <button className="btn btn-link" disabled={true}>
          <i
            className="fas fa-couch"
            style={{ color: "orange", fontSize: "20px" }}
          ></i>
        </button>
        <label>Infant</label>
      </div>
      <div
        className="col"
        style={!checkinPassenger ? { display: "" } : { display: "none" }}
      >
        <button className="btn btn-link" disabled={true}>
          <i
            className="fas fa-couch"
            style={{ color: "brown", fontSize: "20px" }}
          ></i>
        </button>
        <label>Meals</label>
      </div>
    </div>
  );
};
export default SeatType;
