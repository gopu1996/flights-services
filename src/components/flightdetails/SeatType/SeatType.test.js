import React from 'react';
import { shallow, mount ,configure } from 'enzyme';
import { spy } from 'sinon';
configure({ adapter: new Adapter() });
import Adapter from "enzyme-adapter-react-16";
import SeatType from './SeatType';

describe('Login Component', () => {
 
   it('renders correctly', () => {
       expect(shallow(<SeatType />)).toMatchSnapshot();
       });

    it('should render without throwing an error', () => {
      expect(shallow(<SeatType />).find('.row').exists()).toBe(true)
    })

    it('renders a email input', () => {
        expect(shallow(<SeatType />).find('.col').length).toEqual(5)
      
       })
   })

  
   