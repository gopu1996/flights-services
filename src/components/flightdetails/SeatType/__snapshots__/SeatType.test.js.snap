// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SeatType />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="col"
          style={
                    Object {
                              "display": "none",
                            }
          }
>
          <button
                    className="btn btn-link"
                    disabled={true}
          >
                    <i
                              className="fas fa-couch"
                              style={
                                        Object {
                                                  "color": "#B3B6B7",
                                                  "fontSize": "20px",
                                                  "marginLeft": "-10px",
                                                }
                              }
                    />
          </button>
          <label>
                    Available
          </label>
</div>,
        <div
          className="col"
          style={
                    Object {
                              "display": "none",
                            }
          }
>
          <button
                    className="btn btn-link"
                    disabled={true}
          >
                    <i
                              className="fas fa-couch"
                              style={
                                        Object {
                                                  "color": "red",
                                                  "fontSize": "20px",
                                                }
                              }
                    />
          </button>
          <label>
                    Booked
          </label>
</div>,
        <div
          className="w-100"
/>,
        <div
          className="col"
          style={
                    Object {
                              "display": "none",
                            }
          }
>
          <button
                    className="btn btn-link"
                    disabled={true}
          >
                    <i
                              className="fas fa-couch"
                              style={
                                        Object {
                                                  "color": "blue",
                                                  "fontSize": "20px",
                                                  "marginLeft": "-10px",
                                                }
                              }
                    />
          </button>
          <label>
                    WheelChair
          </label>
</div>,
        <div
          className="col"
          style={
                    Object {
                              "display": "none",
                            }
          }
>
          <button
                    className="btn btn-link"
                    disabled={true}
          >
                    <i
                              className="fas fa-couch"
                              style={
                                        Object {
                                                  "color": "orange",
                                                  "fontSize": "20px",
                                                }
                              }
                    />
          </button>
          <label>
                    Infant
          </label>
</div>,
        <div
          className="col"
          style={
                    Object {
                              "display": "",
                            }
          }
>
          <button
                    className="btn btn-link"
                    disabled={true}
          >
                    <i
                              className="fas fa-couch"
                              style={
                                        Object {
                                                  "color": "brown",
                                                  "fontSize": "20px",
                                                }
                              }
                    />
          </button>
          <label>
                    Meals
          </label>
</div>,
      ],
      "className": "row",
      "style": Object {
        "marginTop": "3px",
        "position": "fixed",
        "width": "310px",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <button
              className="btn btn-link"
              disabled={true}
>
              <i
                            className="fas fa-couch"
                            style={
                                          Object {
                                                        "color": "#B3B6B7",
                                                        "fontSize": "20px",
                                                        "marginLeft": "-10px",
                                                      }
                            }
              />
</button>,
            <label>
              Available
</label>,
          ],
          "className": "col",
          "style": Object {
            "display": "none",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-couch"
                style={
                                Object {
                                                "color": "#B3B6B7",
                                                "fontSize": "20px",
                                                "marginLeft": "-10px",
                                              }
                }
/>,
              "className": "btn btn-link",
              "disabled": true,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-couch",
                "style": Object {
                  "color": "#B3B6B7",
                  "fontSize": "20px",
                  "marginLeft": "-10px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Available",
            },
            "ref": null,
            "rendered": "Available",
            "type": "label",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <button
              className="btn btn-link"
              disabled={true}
>
              <i
                            className="fas fa-couch"
                            style={
                                          Object {
                                                        "color": "red",
                                                        "fontSize": "20px",
                                                      }
                            }
              />
</button>,
            <label>
              Booked
</label>,
          ],
          "className": "col",
          "style": Object {
            "display": "none",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-couch"
                style={
                                Object {
                                                "color": "red",
                                                "fontSize": "20px",
                                              }
                }
/>,
              "className": "btn btn-link",
              "disabled": true,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-couch",
                "style": Object {
                  "color": "red",
                  "fontSize": "20px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Booked",
            },
            "ref": null,
            "rendered": "Booked",
            "type": "label",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "w-100",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <button
              className="btn btn-link"
              disabled={true}
>
              <i
                            className="fas fa-couch"
                            style={
                                          Object {
                                                        "color": "blue",
                                                        "fontSize": "20px",
                                                        "marginLeft": "-10px",
                                                      }
                            }
              />
</button>,
            <label>
              WheelChair
</label>,
          ],
          "className": "col",
          "style": Object {
            "display": "none",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-couch"
                style={
                                Object {
                                                "color": "blue",
                                                "fontSize": "20px",
                                                "marginLeft": "-10px",
                                              }
                }
/>,
              "className": "btn btn-link",
              "disabled": true,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-couch",
                "style": Object {
                  "color": "blue",
                  "fontSize": "20px",
                  "marginLeft": "-10px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "WheelChair",
            },
            "ref": null,
            "rendered": "WheelChair",
            "type": "label",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <button
              className="btn btn-link"
              disabled={true}
>
              <i
                            className="fas fa-couch"
                            style={
                                          Object {
                                                        "color": "orange",
                                                        "fontSize": "20px",
                                                      }
                            }
              />
</button>,
            <label>
              Infant
</label>,
          ],
          "className": "col",
          "style": Object {
            "display": "none",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-couch"
                style={
                                Object {
                                                "color": "orange",
                                                "fontSize": "20px",
                                              }
                }
/>,
              "className": "btn btn-link",
              "disabled": true,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-couch",
                "style": Object {
                  "color": "orange",
                  "fontSize": "20px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Infant",
            },
            "ref": null,
            "rendered": "Infant",
            "type": "label",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <button
              className="btn btn-link"
              disabled={true}
>
              <i
                            className="fas fa-couch"
                            style={
                                          Object {
                                                        "color": "brown",
                                                        "fontSize": "20px",
                                                      }
                            }
              />
</button>,
            <label>
              Meals
</label>,
          ],
          "className": "col",
          "style": Object {
            "display": "",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-couch"
                style={
                                Object {
                                                "color": "brown",
                                                "fontSize": "20px",
                                              }
                }
/>,
              "className": "btn btn-link",
              "disabled": true,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-couch",
                "style": Object {
                  "color": "brown",
                  "fontSize": "20px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Meals",
            },
            "ref": null,
            "rendered": "Meals",
            "type": "label",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col"
            style={
                        Object {
                                    "display": "none",
                                  }
            }
>
            <button
                        className="btn btn-link"
                        disabled={true}
            >
                        <i
                                    className="fas fa-couch"
                                    style={
                                                Object {
                                                            "color": "#B3B6B7",
                                                            "fontSize": "20px",
                                                            "marginLeft": "-10px",
                                                          }
                                    }
                        />
            </button>
            <label>
                        Available
            </label>
</div>,
          <div
            className="col"
            style={
                        Object {
                                    "display": "none",
                                  }
            }
>
            <button
                        className="btn btn-link"
                        disabled={true}
            >
                        <i
                                    className="fas fa-couch"
                                    style={
                                                Object {
                                                            "color": "red",
                                                            "fontSize": "20px",
                                                          }
                                    }
                        />
            </button>
            <label>
                        Booked
            </label>
</div>,
          <div
            className="w-100"
/>,
          <div
            className="col"
            style={
                        Object {
                                    "display": "none",
                                  }
            }
>
            <button
                        className="btn btn-link"
                        disabled={true}
            >
                        <i
                                    className="fas fa-couch"
                                    style={
                                                Object {
                                                            "color": "blue",
                                                            "fontSize": "20px",
                                                            "marginLeft": "-10px",
                                                          }
                                    }
                        />
            </button>
            <label>
                        WheelChair
            </label>
</div>,
          <div
            className="col"
            style={
                        Object {
                                    "display": "none",
                                  }
            }
>
            <button
                        className="btn btn-link"
                        disabled={true}
            >
                        <i
                                    className="fas fa-couch"
                                    style={
                                                Object {
                                                            "color": "orange",
                                                            "fontSize": "20px",
                                                          }
                                    }
                        />
            </button>
            <label>
                        Infant
            </label>
</div>,
          <div
            className="col"
            style={
                        Object {
                                    "display": "",
                                  }
            }
>
            <button
                        className="btn btn-link"
                        disabled={true}
            >
                        <i
                                    className="fas fa-couch"
                                    style={
                                                Object {
                                                            "color": "brown",
                                                            "fontSize": "20px",
                                                          }
                                    }
                        />
            </button>
            <label>
                        Meals
            </label>
</div>,
        ],
        "className": "row",
        "style": Object {
          "marginTop": "3px",
          "position": "fixed",
          "width": "310px",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="btn btn-link"
                disabled={true}
>
                <i
                                className="fas fa-couch"
                                style={
                                                Object {
                                                                "color": "#B3B6B7",
                                                                "fontSize": "20px",
                                                                "marginLeft": "-10px",
                                                              }
                                }
                />
</button>,
              <label>
                Available
</label>,
            ],
            "className": "col",
            "style": Object {
              "display": "none",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fas fa-couch"
                  style={
                                    Object {
                                                      "color": "#B3B6B7",
                                                      "fontSize": "20px",
                                                      "marginLeft": "-10px",
                                                    }
                  }
/>,
                "className": "btn btn-link",
                "disabled": true,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-couch",
                  "style": Object {
                    "color": "#B3B6B7",
                    "fontSize": "20px",
                    "marginLeft": "-10px",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Available",
              },
              "ref": null,
              "rendered": "Available",
              "type": "label",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="btn btn-link"
                disabled={true}
>
                <i
                                className="fas fa-couch"
                                style={
                                                Object {
                                                                "color": "red",
                                                                "fontSize": "20px",
                                                              }
                                }
                />
</button>,
              <label>
                Booked
</label>,
            ],
            "className": "col",
            "style": Object {
              "display": "none",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fas fa-couch"
                  style={
                                    Object {
                                                      "color": "red",
                                                      "fontSize": "20px",
                                                    }
                  }
/>,
                "className": "btn btn-link",
                "disabled": true,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-couch",
                  "style": Object {
                    "color": "red",
                    "fontSize": "20px",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Booked",
              },
              "ref": null,
              "rendered": "Booked",
              "type": "label",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "w-100",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="btn btn-link"
                disabled={true}
>
                <i
                                className="fas fa-couch"
                                style={
                                                Object {
                                                                "color": "blue",
                                                                "fontSize": "20px",
                                                                "marginLeft": "-10px",
                                                              }
                                }
                />
</button>,
              <label>
                WheelChair
</label>,
            ],
            "className": "col",
            "style": Object {
              "display": "none",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fas fa-couch"
                  style={
                                    Object {
                                                      "color": "blue",
                                                      "fontSize": "20px",
                                                      "marginLeft": "-10px",
                                                    }
                  }
/>,
                "className": "btn btn-link",
                "disabled": true,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-couch",
                  "style": Object {
                    "color": "blue",
                    "fontSize": "20px",
                    "marginLeft": "-10px",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "WheelChair",
              },
              "ref": null,
              "rendered": "WheelChair",
              "type": "label",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="btn btn-link"
                disabled={true}
>
                <i
                                className="fas fa-couch"
                                style={
                                                Object {
                                                                "color": "orange",
                                                                "fontSize": "20px",
                                                              }
                                }
                />
</button>,
              <label>
                Infant
</label>,
            ],
            "className": "col",
            "style": Object {
              "display": "none",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fas fa-couch"
                  style={
                                    Object {
                                                      "color": "orange",
                                                      "fontSize": "20px",
                                                    }
                  }
/>,
                "className": "btn btn-link",
                "disabled": true,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-couch",
                  "style": Object {
                    "color": "orange",
                    "fontSize": "20px",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Infant",
              },
              "ref": null,
              "rendered": "Infant",
              "type": "label",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="btn btn-link"
                disabled={true}
>
                <i
                                className="fas fa-couch"
                                style={
                                                Object {
                                                                "color": "brown",
                                                                "fontSize": "20px",
                                                              }
                                }
                />
</button>,
              <label>
                Meals
</label>,
            ],
            "className": "col",
            "style": Object {
              "display": "",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <i
                  className="fas fa-couch"
                  style={
                                    Object {
                                                      "color": "brown",
                                                      "fontSize": "20px",
                                                    }
                  }
/>,
                "className": "btn btn-link",
                "disabled": true,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-couch",
                  "style": Object {
                    "color": "brown",
                    "fontSize": "20px",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Meals",
              },
              "ref": null,
              "rendered": "Meals",
              "type": "label",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
